components/director/internal/domain/runtime/converter.go

```diff
@@ -1,4 +1,82 @@
 package runtime
 
-type Converter struct {
+import (
+	"github.com/kyma-incubator/compass/components/director/internal/graphql"
+	"github.com/kyma-incubator/compass/components/director/internal/model"
+)
+
+type Converter struct{}
+
+func (c *Converter) ToGraphQL(in *model.Runtime) *graphql.Runtime {
+	if in == nil {
+		return nil
+	}
+
+	return &graphql.Runtime{
+		ID:          in.ID,
+		Status:      c.statusToGraphQL(in.Status),
+		Name:        in.Name,
+		Description: in.Description,
+		Tenant:      graphql.Tenant(in.Tenant),
+		Annotations: in.Annotations,
+		Labels:      in.Labels,
+	}
+}
+
+func (c *Converter) MultipleToGraphQL(in []*model.Runtime) []*graphql.Runtime {
+	var runtimes []*graphql.Runtime
+	for _, r := range in {
+		if r == nil {
```

Comment:
this `if` is not needed IMO

Reply:
It is, because in GraphQL schema we defined that we won't return `null` in the output array